tests/nvmeof/test_ceph_nvmeof_gateway_sub_scale.py

```diff
@@ -95,26 +95,31 @@ def configure_subsystem(config, _cls, command):
         config["args"].clear()
 
 
-def configure_listener(config, _cls, command, ceph_cluster, node):
+def configure_listener(config, _cls, command, ceph_cluster):
```

Comment:
make it as `configure_listeners`

Reply:
@sunilkumarn417 plural holds good for subsystem, host and namespaces too. So why only listeners. I am planning to change all

Reply:
changed it